src/typography/Typography.tsx

```diff
@@ -14,25 +14,24 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import React, { forwardRef } from "react";
+import React, { forwardRef, ReactNode } from "react";
```

Comment:
Have you made sure there are no uses of these components where we don't use the default value of `as`?